packages/core/index.d.ts

```diff
@@ -143,7 +143,7 @@ declare module '@rjsf/core' {
 
     export type Field = React.StatelessComponent<FieldProps> | React.ComponentClass<FieldProps>;
 
-    export type FieldTemplateProps = {
+    export type FieldTemplateProps<T = any> = {
```

Comment:
What's the difference between doing `<T = any>` and `<T>`? Asking because we have `FormProps<T>` elsewhere in this typings file.

Reply:
I followed the other `FieldTemplateProps` as examples. So for instance `ArrayFieldTemplateProps` and `ObjectFieldTemplateProps`. As I understand it, it defaults the type of `any` so the type doesn't need to be explicitly provided. For example, in the [TS Playground](https://www.typescriptlang.org/play?#code/FAFwngDgpgBAYgSygGwCYBUoFsLIIYhQAKATgPYQDOAPOgHwwC8MA3sDDAGZklYAiBPAC4Y6ANzAAvsGABjMgDtKILkjSYc+QiMQoM2XAWLkq1AMogSCBQHMGzNh268BIYTABEACQQepM+SUVTjV9TSMAeSwEEB1QjUNCUgpKJlZ2Lh5+QRFvX39QSFhddQMtYxSAQQUwWjS8Gvt0pyzXd3ECwOVVPQTy6rA43rKjZKoB80trOzTHTJcczx8-aTlFbpDh8MIBqJih0u2K8ZrZjOdst1zlgqA).